netty/src/main/java/io/grpc/transport/netty/NettyClientHandler.java

```diff
@@ -137,15 +137,16 @@ public void channelActive(ChannelHandlerContext ctx) throws Exception {
   @Override
   public void write(ChannelHandlerContext ctx, Object msg, ChannelPromise promise) {
     try {
-      if (msg instanceof CreateStreamCommand) {
-        createStream((CreateStreamCommand) msg, promise);
+      if (msg instanceof CreateStreamNettyCommand) {
```

Comment:
All of the commands are 'outbound' in the sense that the cause writes so I'm not sure theres an improvement to naming in that approach.